Library/Homebrew/tap.rb

```diff
@@ -49,6 +49,11 @@ def remote
     end
   end
 
+  # True if this {Tap} is a git repository.
+  def git_remote?
```

Comment:
I find this naming confusing because the method don’t really tests if this is a git repo with a remote, just that it’s a git repo. Why not `git?`?